### DISCUSSION TITLE
Why we can't reference a user directly in the dashboard?

### DISCUSSION THREAD

**Discussion by hallee9000** [supabase/supabase#2345] [Questions] [answered]
I just want to create a column which references a user, but I have to write some SQL in the editor to do this. Why just do it in the foreign key selector?

![image](https://user-images.githubusercontent.com/12868055/125442948-683a1a13-fecf-4371-9106-d604403aa83a.png)

Also if I want to add some extra information to the user, the recommended way is cloning a public user table and sync it after new user added. I think it would be better if we can extend auth user table.

**Comment by RaghavBhat02** [comment]
I am having the same issue.

**Comment by inian** [comment, marked as answer]
The users table is in the `auth` schema which is not reference-able at this point. We have a issue for this [here](https://github.com/supabase/supabase/issues/1129)
